content/articles/How-to-start-up-with-Socket-Programming-with-Java/index.md

```diff
@@ -104,12 +105,12 @@ while (true) {
    
 ### Methods used in ServerSocket
 
--` public InputStream( getInputStream())` - returns an InputStream object that receives data.
-- `public OutputStream( getOutputStream())`- returns an OutputStream for sending data.
-- `public Socket accept ( )` -Waits for client connection( the program won't move to the next level until the client is connected). When you connect, you'll get a socket object that you may use to communicate with the client. 
--  void close ( ) - closes the server socket i.e stops waiting for requests from clients.
+-`public InputStream getInputStream()` - returns an InputStream object that receives data.
+-`public OutputStream getOutputStream()`- returns an OutputStream for sending data.
+-`public Socket accept()` -Waits for client connection( the program won't continue until the client is connected). When you connect, you'll get a socket object that you may use to communicate with the client. 
```

Comment:
Change:

>-`public Socket accept()` -Waits for client connection( the program won't continue until the client is connected). When you connect, you'll get a socket object that you may use to communicate with the client. 
-`void close ()` - closes the server socket i.e stops waiting for requests from clients.\

To:

>-`public Socket accept()` - Waits for client connection (the program won't continue until the client is connected). When they connect, you'll get a socket object that you may use to communicate with the client.
-`void close()` - closes the server socket i.e stops waiting for requests from clients.

Only fixing some misplaced/missing spaces